src/MicroserviceErrorHandler.js

```diff
@@ -36,8 +36,19 @@ module.exports = class MicroserviceErrorHandler {
 
     createErrorHandler() {
         return (err, req, res, next) => {
-            // eslint-disable-next-line no-param-reassign
-            err.serviceName = this.serviceName;
+            // prevent the service from overriding errors from remote services
+            if (!err.serviceName) {
+                // eslint-disable-next-line no-param-reassign
```

Comment:
should we remove this rule from our linter config?
In my opinion it often makes sense to reassign the param